contributing.md

```diff
@@ -95,6 +97,39 @@ We're a UNIX-friendly organization; we love making lots of small modules that do
 
 If you're creating a new repository, please follow the new repository guidelines [orbitdb/repo-template](https://github.com/orbitdb/repo-template).
 
+#### Linting
+
+To add [`standard`](https://standardjs.com/) to a repository:
+
+```sh
+npm i -D standard # npm install --save-dev standard
+```
+
+Manually add `"lint": "standard --env=mocha"` to your `package.json` `.scripts` field. Then, run `standard --fix` to fix current issues, and `npm run lint` to show the output of the linter.
+
+If you'd rather have the environment set permanently (for instance, so that your editor can notice it), add this to the `package.json`:
+
+```json
+{
+  "standard": {
+    "env": [
+      "mocha"
+    ]
+  }
+}
+```
```

Comment:
Consider separating code and instructions, and adding discrete steps with imperative language.

#### Linting

When contributing code, please "lint first and ask questions later." We use https://standardjs.com to lint our code.
 
1. Install [`standard`](https://standardjs.com/):

```sh
$ npm i -D standard # npm install --save-dev standard
```
2. Add the `lint` script to package.json
```json
{
  "scripts": {
    "lint": "standard --env=mocha"
  }
}
```
3. Run `standard --fix` to fix current issues
```sh
$ standard --fix
```
4. Run the linter
```
$ npm run lint
```

***Note:*** If you'd rather have the environment set permanently (for instance, so that your editor can notice it), add this to the `package.json`:
 ```json
{
  "standard": {
    "env": [
      "mocha"
    ]
  }
}
```

Reply:
Added `npx` to the `npx standard --fix` command

Reply:
Also, we might want to add a note here about the `exclude` option - I ran into this when implementing `standard` in `ipfs-log`

Reply:
Good idea. 

Do you need to install with npx, too?

Reply:
negative, npx is only for running the executables in `./node_modules/.bin`

Reply:
How's it look now?

Reply:
LGTM :+1: